### PR TITLE
Add test cases for soul-admin EnumService

### PR DESCRIPTION
**Pull request by onlyonezhongjinhui**

### CODE REVIEW

**Comment by songyuequan** [comment]
good job

**Review by songyuequan — APPROVED** [review]

File: soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+@RunWith(MockitoJUnitRunner.class)
+public final class EnumServiceTest {
+
+    @InjectMocks
+    private EnumServiceImpl enumService;
+
+    @Test
+    public void testListSize() {
+        Map<String, List<EnumVO>> list = enumService.list();
+        assertEquals(13, list.size());
```
Comment (dengliming) [inline soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java:45]: greater than zero is enough.

File: soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    @Test
+    public void testListEle() {
+        Map<String, List<EnumVO>> list = enumService.list();
+        assertNotNull(list.get("httpMethodEnums"));
```
Comment (dengliming) [inline soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java:51]: Can we also verify the the elements in HttpMethodEnum? Similar in other places.

File: soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java (hunk 1 above)
Comment (onlyonezhongjinhui) [reply soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java:45]: ok

File: soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java (hunk 2 above)
Comment (onlyonezhongjinhui) [reply soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java]: right

File: soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+@RunWith(MockitoJUnitRunner.class)
+public final class EnumServiceTest {
+
+    @InjectMocks
+    private EnumServiceImpl enumService;
+
+    @Test
+    public void testListSize() {
+        Map<String, List<EnumVO>> list = enumService.list();
+        assertTrue(list.size() > 0);
```
Comment (dengliming) [inline soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java:45]: Can you use `hamcrest`? `assertThat(list.size(), greaterThan(0));`

Comment (onlyonezhongjinhui) [reply soul-admin/src/test/java/org/dromara/soul/admin/service/EnumServiceTest.java]: ok

**Review by dengliming — COMMENTED** [review]
Looks good. Here's a little advice. Most of your test logic is the same except for ` list.get(key)`, I suggest you use `@ParameterizedTest`, Here is a simple example.
<details>
<summary>Click to expand</summary>

```
private EnumServiceImpl enumService = new EnumServiceImpl();

    @ParameterizedTest(name = "{index} => test {0}")
    @ValueSource(strings = { "hystrixIsolationModeEnums" })
    public void testListEnums(String key) {
        Map<String, List<EnumVO>> list = enumService.list();
        assertThat(list, hasKey(key));

        List<EnumVO> enums = list.get(key);
        assertThat(enums.size(), greaterThan(0));
        enums.forEach(e -> assertThat(e.getSupport(), comparesEqualTo(true)));
    }
```
</details>

**Review by dengliming — APPROVED** [review]
LGTM

**Pull request merged**